packages/example-browser/webpack.config.js

```diff
@@ -43,7 +43,7 @@ const common = {
 if (process.env['NODE_ENV'] === 'production') {
     module.exports = merge(common, {
         plugins: [
-            new UglifyJSPlugin(),
+            new terser(),
```

Comment:
nitpicking: class shouldn't start with a capital letter?

btw, with webpack5, i'm not sure we need to add Terser plugin by hands it we don't need to overrides options

Reply:
All webpack config are gone.